pkg/util/secretutil/secret.go

```diff
@@ -151,27 +152,30 @@ func ValidateSecretObject(secretObj v1alpha1.SecretObject) error {
 func GetSecretData(secretObjData []*v1alpha1.SecretObjectData, secretType corev1.SecretType, files map[string]string) (map[string][]byte, error) {
 	datamap := make(map[string][]byte)
 	for _, data := range secretObjData {
-		if len(data.ObjectName) == 0 {
+		objectName := strings.TrimSpace(data.ObjectName)
+		dataKey := strings.TrimSpace(data.Key)
+
+		if len(objectName) == 0 {
 			return datamap, fmt.Errorf("object name in secretObjects.data")
 		}
-		if len(data.Key) == 0 {
+		if len(dataKey) == 0 {
 			return datamap, fmt.Errorf("key in secretObjects.data is empty")
 		}
-		file, ok := files[data.ObjectName]
+		file, ok := files[strings.TrimSpace(objectName)]
```

Comment:
why do you need to trim again here?

Reply:
s/`data.ObjectName`/`objectName` 🤦 

Updated it!